pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -223,4 +224,7 @@ func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfigura
 	if obj.EnforceNodeAllocatable == nil {
 		obj.EnforceNodeAllocatable = DefaultNodeAllocatableEnforcement
 	}
+	if obj.RegistryCredentialConfigPath == "" {
```

Comment:
Do we need this? If its not set, wouldn't it be better to leave it not set?

Reply:
Yeah, makes sense for alpha.

Reply:
The issue I've run into here, is that now that I've removed the default, there is no obvious way to set it from local-up-cluster.sh. I will have to add a KubeletConfiguration file there--do you think I should do that, or is a default value ok here?